src/components/SecondaryPanes/Frames/index.js

```diff
@@ -84,13 +95,15 @@ class Frames extends Component {
           frameOrGroup.id
             ? FrameComponent({
                 frame: frameOrGroup,
+                copyStackTrace: () => this.copyStackTrace(),
```

Comment:
did u guys consider using [property initializers](https://babeljs.io/docs/plugins/transform-class-properties/)? `this` problem would go away ;)